assets/js/modules/analytics/components/common/PropertySelect.test.js

```diff
@@ -95,14 +95,16 @@ describe( 'PropertySelect', () => {
 			.toHaveClass( 'mdc-select--disabled' );
 	} );
 
-	it( 'should be disabled when in the absence of an valid account ID.', async () => {
+	it( 'should not render in the absence of an valid account ID.', async () => {
```

Comment:
Again: I know this wasn't your wording but let's take the time to simplify it:

```suggestion
	it( 'should not render if account ID is invalid', async () => {
```